### PR TITLE
CYS -- Intro design updates

### PR DESCRIPTION
**Pull request by moon0326**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

This PR updates the CYS intro page to reflex the latest design changes.

1. Replaced the banner with the latest one
2. Added banner and text copy for offline mode
3. Various style fixes
4. Added `Active theme` badge
5. Added color color palettes

This PR doesn't handle theme click event, but `ThemeCard` component accepts `link` prop. Once we have changes from https://github.com/woocommerce/woocommerce/issues/40482, we should refactor the code. 

This PR doesn't handle Jetpack connection design yet. You can still test it by flipping https://github.com/woocommerce/woocommerce/pull/40503/files#diff-0c89278189ca36a2aa455c6193226c3abd4b76a4bb5a78d692ea8c11b2bad495R40 variable

### How to test the changes in this Pull Request:

This feature is part of the larger customize your store development and behind a feature flag, hence QA review can be deferred until the call for testing before release.

Make sure to install & active WooCommerce Beta Tester and turn on `customize-store` feature flag from `Tools -> WCA Test Helper -> Features`

1. Create a new JN site with this branch.
2. Complete the core profiler
3. Go to `WooCommerce Home` and click on `Customize your store` task
4. Turn off your WIFI
5. You should see the offline banner and copy.

Screen Shot 2023-09-28 at 2 05 23 PM
6. Compare the page with the Figma and confirm the design matches.

### Changelog entry

-   [x] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [ ] Patch
-   [x] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [x] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 

CYS - Intro page design updates

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce-admin/client/customize-store/intro/intro.scss [hunk 1]
```diff
@@ -12,14 +12,28 @@
...
+	.edit-site-site-icon {
+		line-height: 0; 
+
+		svg {
+			height: 32px;
+			width: 32px;
+		}
+	}
+
+	.edit-site-site-hub__site-title {
```
Comment (chihsuan) [inline plugins/woocommerce-admin/client/customize-store/intro/intro.scss:30]: Minor UI issue

Screenshot 2023-09-29 at 13 05 24

```
color: $gray-900;
```

**Review by chihsuan — COMMENTED** [review]
Thanks for working on this! 👍  @moon0326 and @crunnells 

I just noticed some UI issues:

1. `margin` between title and description should be 8px;

Acutal:
Screenshot 2023-09-29 at 13 10 45

Expected
Screenshot 2023-09-29 at 13 12 20

2. The banner description margin-top should be `12px` and margin-bottom should be `28px`. Button height should be `40px`;

Actual
Screenshot 2023-09-29 at 13 14 42

Expected
Screenshot 2023-09-29 at 13 15 07

3. Theme card grid vertical gap between rows should be `54px`;

Actual:

Screenshot 2023-09-29 at 13 20 59


Expected:

Screenshot 2023-09-29 at 13 20 26

**Review by adrianduffell — APPROVED** [review]
This is great @moon0326 , thank you! 🚀 

Approving, but it would be great to have @chihsuan's suggestions made. I also [found](https://github.com/woocommerce/woocommerce/issues/40482#issuecomment-1740490532) it might be possible to use transparency instead of setting border colors but I'm happy for that to be explored in a follow-up 😄

**Comment by moon0326** [comment]
> This is great @moon0326 , thank you! 🚀
> 
> Approving, but it would be great to have @chihsuan's suggestions made. I also https://github.com/woocommerce/woocommerce/issues/40482#issuecomment-1740490532 it might be possible to use transparency instead of setting border colors but I'm happy for that to be explored in a follow-up 😄

> This is great @moon0326 , thank you! 🚀
> 
> Approving, but it would be great to have @chihsuan's suggestions made. I also https://github.com/woocommerce/woocommerce/issues/40482#issuecomment-1740490532 it might be possible to use transparency instead of setting border colors but I'm happy for that to be explored in a follow-up 😄

Thank you! TIL! :) Updated in https://github.com/woocommerce/woocommerce/pull/40503/commits/b2fac99d33e81c4f095dc3649b3f01076b7e66f4https://github.com/woocommerce/woocommerce/pull/40503/commits/b2fac99d33e81c4f095dc3649b3f01076b7e66f4

File: plugins/woocommerce-admin/client/customize-store/intro/intro.scss (hunk 1 above)
Comment (moon0326) [reply plugins/woocommerce-admin/client/customize-store/intro/intro.scss:30]: Updated in https://github.com/woocommerce/woocommerce/pull/40503/commits/926e5a398e66963b8f93d4e166a755f47bd8214c

Comment (moon0326) [reply plugins/woocommerce-admin/client/customize-store/intro/intro.scss:30]: Updated other CSS issues in https://github.com/woocommerce/woocommerce/pull/40503/commits/49a3530a188e51ade32174283014c4e161200bc0 and [Update theme card row-gap](https://github.com/woocommerce/woocommerce/pull/40503/commits/3e1abf2a90d065301eaccdd029180a95fab25cbf)

**Pull request merged**